rust/operator-binary/src/main.rs

```diff
@@ -76,15 +76,12 @@ async fn main() -> anyhow::Result<()> {
                 built_info::BUILT_TIME_UTC,
                 built_info::RUSTC_VERSION,
             );
-            let controller_config = ControllerConfig {
-                broker_clusterrole: kafka_broker_clusterrole,
-            };
             let product_config = product_config.load(&[
                 "deploy/config-spec/properties.yaml",
                 "/etc/stackable/kafka-operator/config-spec/properties.yaml",
             ])?;
             let client = client::create_client(Some(OPERATOR_NAME.to_string())).await?;
-            create_controller(client, controller_config, product_config, watch_namespace).await;
+            create_controller(client, product_config, watch_namespace).await;
```

Comment:
How did the create_controller signature change from just upgrade rust?

Reply:
The argument passed in here was used as a struct field which was unused.

Reply:
I also added a changelog entry.